packages/components-css/package.json

```diff
@@ -1,11 +1,11 @@
 {
   "version": "1.0.0-alpha.0",
-  "author": "Community for NL Design System",
+  "author": "Frameless",
   "description": "CSS Components for a design system based on the NL Design System architecture",
   "license": "EUPL-1.2",
-  "name": "@example/components-css",
+  "name": "@frameless/components-css",
   "keywords": [
-    "nl-design-system"
+    "frameless"
```

Comment:
Ik zou nl design system laten staan. Dan kun je bij npm vinden wat allemaal packages zijn die iets met nl design system te maken hebben. We baseren onze architectuur immers daar op?